diff --git a/dist/index.d.ts b/dist/index.d.ts
index eee27438909b5b12efee4f42f443d14b20275166..4a8cb65c74400386eb3ab333f8fdf63a6db5cd3e 100644
--- a/dist/index.d.ts
+++ b/dist/index.d.ts
@@ -78,8 +78,8 @@ export declare namespace OpenAPIV3_1 {
     }
     export type BaseSchemaObject = Modify<Omit<OpenAPIV3.BaseSchemaObject, 'nullable'>, {
         examples?: OpenAPIV3.BaseSchemaObject['example'][];
-        exclusiveMinimum?: boolean | number;
-        exclusiveMaximum?: boolean | number;
+        exclusiveMinimum?: number;
+        exclusiveMaximum?: number;
         contentMediaType?: string;
         $schema?: string;
         additionalProperties?: boolean | ReferenceObject | SchemaObject;
@@ -273,9 +273,9 @@ export declare namespace OpenAPIV3 {
         default?: any;
         multipleOf?: number;
         maximum?: number;
-        exclusiveMaximum?: boolean;
+        exclusiveMaximum?: number;
         minimum?: number;
-        exclusiveMinimum?: boolean;
+        exclusiveMinimum?: number;
         maxLength?: number;
         minLength?: number;
         pattern?: string;
@@ -652,9 +652,9 @@ export declare namespace OpenAPIV2 {
         collectionFormat?: string;
         default?: any;
         maximum?: number;
-        exclusiveMaximum?: boolean;
+        exclusiveMaximum?: number;
         minimum?: number;
-        exclusiveMinimum?: boolean;
+        exclusiveMinimum?: number;
         maxLength?: number;
         minLength?: number;
         pattern?: string;
@@ -698,9 +698,9 @@ export interface IJsonSchema {
     description?: string;
     multipleOf?: number;
     maximum?: number;
-    exclusiveMaximum?: boolean;
+    exclusiveMaximum?: number;
     minimum?: number;
-    exclusiveMinimum?: boolean;
+    exclusiveMinimum?: number;
     maxLength?: number;
     minLength?: number;
     pattern?: string;
